src/bundle/Resources/doc/FEATURES.md

```diff
@@ -0,0 +1,18 @@
+# Features
```

Comment:
TBH the whole doc section should be dropped and moved to the official documentation. This has been a pattern for quite some time.